pkg/kubelet/secret_manager.go

```diff
@@ -39,3 +54,181 @@ func newSimpleSecretManager(kubeClient clientset.Interface) (secretManager, erro
 func (s *simpleSecretManager) GetSecret(namespace, name string) (*api.Secret, error) {
 	return s.kubeClient.Core().Secrets(namespace).Get(name)
 }
+
+func (s *simpleSecretManager) RegisterPod(pod *api.Pod) {
+}
+
+func (s *simpleSecretManager) UnregisterPod(pod *api.Pod) {
+}
+
+// secretStoreItems is a single item stored in secretStore.
+type secretStoreItem struct {
+	secret   *api.Secret
+	refCount int
+}
+
+// secretStore is a local cache of secrets.
+type secretStore struct {
+	kubeClient clientset.Interface
+
+	sync.Mutex
+	items map[string]*secretStoreItem
```

Comment:
make the key a `struct{namespace,name string}` to avoid continual construction of new key strings?